.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!-- Thank you for submitting a pull request! -->
+
+**Summary**
+<!-- What kind of change does this PR introduce? -->
+<!-- Is this a bugfix, feature, refactoring, build related change, etc? -->
+
+<!-- Describe the need for this change -->
+
+<!-- Link any documentation or information that would help understand this change -->
+
+**Other information**
```

Comment:
WDYT about changing this into `**Related Issues/PRs**` we frequently end up linking blocking PRs/closing issues/etc

Other seems like something one can always add on their own :)

Reply:
Sounds good to me!